examples/__init__.py

```diff
@@ -3,4 +3,5 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+"""Initialises examples package."""
```

Comment:
What's with all the "initialize" and "initializes" comments? Redundant because it's obvious imo. Therefore violates DRY